symengine/tests/cwrapper/test_cwrapper.c

```diff
@@ -1013,7 +1118,8 @@ int main(int argc, char *argv[])
 #endif // HAVE_SYMENGINE_MPFR
 #ifdef HAVE_SYMENGINE_MPC
     test_complex_mpc();
-#endif // HAVE_SYMENGINE_MPC
-
+#endif // HAVE_SYMENGINE_MPC*/
+    symengine_print_stack_on_segfault();
+    test_matrix();
```

Comment:
I'll uncomment the lines, once all the tests are done.
Commented them out, so running the tests would be easier locally.